packages/map-template/src/components/Search/Search.jsx

```diff
@@ -31,9 +34,13 @@ function Search({ onSetSize }) {
     /** Referencing the search field */
     const searchFieldRef = useRef();
 
+    /** Referencing the keyboard element */
+    const keyboardRef = useRef();
+
     const [searchDisabled, setSearchDisabled] = useState(true);
     const [searchResults, setSearchResults] = useState([]);
     const categories = useRecoilValue(categoriesState);
+    const showKeyboard = useRecoilValue(showKeyboardState);
```

Comment:
showKeyboard can also mean that it is currently shown. Should the atom (and even the prop) be renamed into something like `useKeyboard`?

Reply:
Good idea, I renamed the atom and the prop.

Reply:
👏